packages/fast-components-react-base/src/tabs/tabs.props.ts

```diff
@@ -0,0 +1,53 @@
+import * as React from "react";
+import { IManagedClasses, ITabsClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+
+export enum Orientation {
+    horizontal = "horizontal",
+    vertical = "vertical"
+}
+
+export interface ITabsHandledProps {
+    /**
+     * The active tab item id
+     */
+    activeId?: string;
+
+    /**
+     * The tabs content
+     */
+    children?: React.ReactNode | React.ReactNode[];
+
+    /**
+     * The aria-label applied to the tablist for the tab items
+     */
+    label: string;
+
+    /**
+     * The tab update callback
```

Comment:
Can you make the `onUpdateTab` comment more descriptive? Why would I want to use this prop? When and why is this function called?